### PR TITLE
Windows share support without using controller publish

### PR DESCRIPTION
**Pull request by pnookala-px**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ggriffiths — APPROVED** [review]
The changes to the API looks good

File: api/spec/spec_handler.go [hunk 1]
```diff
@@ -606,6 +607,12 @@ func (d *specHandler) UpdateSpecFromOpts(opts map[string]string, spec *api.Volum
 			} else {
 				spec.FpPreference = fastpath
 			}
+		case api.SpecWinshare:
+			if winshare, err := strconv.ParseBool(v); err != nil {
+				return nil, nil, nil, err
+			} else {
+				spec.Winshare = winshare
+			}
```
Comment (sulakshm) [inline api/spec/spec_handler.go:615]: else is not needed after return.

**Review by sulakshm — APPROVED** [review]
why isnt the changes still not merged?

File: api/spec/spec_handler.go (hunk 1 above)
Comment (pnookala-px) [reply api/spec/spec_handler.go:615]: but that is how code is written in all places IN THIS FUNCTION.

**Pull request merged**